src/Nethermind/Nethermind.Trie.Test/TrieTests.cs

```diff
@@ -1031,16 +1033,16 @@ public void When_two_branches_with_two_same_children_change_one_and_change_back_
                                 stateProvider.IncrementNonce(address);
                             }
 
-                            byte[] storage = new byte[1];
-                            _random.NextBytes(storage);
+                            UInt256 storage = UInt256.One;
+                            _random.NextBytes(MemoryMarshal.AsBytes(MemoryMarshal.CreateSpan(ref storage, 1)));
                             storageProvider.Set(new StorageCell(address, 1), storage);
                         }
                         else if (!account.IsTotallyEmpty)
                         {
                             stateProvider.CreateAccount(address, account.Balance);
 
-                            byte[] storage = new byte[1];
-                            _random.NextBytes(storage);
+                            UInt256 storage = UInt256.One;
+                            _random.NextBytes(MemoryMarshal.AsBytes(MemoryMarshal.CreateSpan(ref storage, 1)));
```

Comment:
Doesn't it create a dependency on knowing how `Uint256` is represented internally? This is a separate package now, and yes, it's quite unlikely that we'll add 8 more bytes to it, but can we random like this? I think I'd prefer to `stackalloc byte[32]`, call random on it and then read from bytes with `Uint256`. Or extend `Random` with an exstension method `NextUint256` that does that.